ghost/email-service/lib/batch-sending-service.js

```diff
@@ -286,6 +286,7 @@ class BatchSendingService {
         try {
             const members = await this.getBatchMembers(batch.id);
             const response = await this.#sendingService.send({
+                emailId: email.id,
```

Comment:
Allows adding reference to email id in mailgun, as currently added in mailgun client [here](https://github.com/TryGhost/Ghost/blob/80decf83031acdab9916d758eaddde08840ba2e8/ghost/mailgun-client/lib/mailgun-client.js#L69-L72)